Phone number assignment cmdlet update

JensTrier opened the pull request:
Added more descriptions to the 2 pages and fixed a typo

opbld33 commented:
Docs Build status updates of commit _[b8dd342](https://github.com/JensTrier/jenstr-office-docs-powershell/commits/b8dd342d08e76b6310ec6a9c310893a7ad51ce63)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/PhoneNumerAssignmentUpdate/teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/teams/remove-csphonenumberassignment?view=teams-ps&branch=pr-en-us-8942) (teams-ps) |
[teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/PhoneNumerAssignmentUpdate/teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/teams/set-csphonenumberassignment?view=teams-ps&branch=pr-en-us-8942) (teams-ps) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C2%5C11%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202202111504044323-8942%5Cworkflow_report.html?sv=2020-08-04&se=2022-03-14T15%3A08%3A50Z&sr=b&sp=r&sig=TKNjYZg5VuJZovfK4moyOiRhfAlOGjIA8ilOI68DG1w%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/3603fd28-efce-f87f-e986-fc8a694fc312?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

JohanFreelancer9 commented on teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md:
```diff
@@ -67,7 +67,7 @@ Accept wildcard characters: False
 
 ### -PhoneNumber
-The phone number to unnassign from the user or resource account. Supports E.164 format and non-E.164 format.
+The phone number to unassign from the user or resource account. Supports E.164 format and non-E.164 format. Needs to be without prefixed "tel:".
```
```suggestion
The phone number to unassign from the user or resource account. Supports E.164 format and non-E.164 format. Needs to be without the prefixed "tel:".

```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md:
```diff
@@ -31,7 +31,9 @@ Set-CsPhoneNumberAssignment -Identity <String> -EnterpriseVoiceEnabled <Boolean>
-This cmdlet assigns a phone number to a user or resource account.
+This cmdlet assigns a phone number to a user or resource account. When you assign a phone number the EnterpriseVoiceEnabled flag is automatically set to True.
+
+To remove a phone number from a user or resource account use the cmdlet [Remove-CsPhoneNumberAssignment](Remove-CsPhoneNumberAssignment.md)
```
```suggestion
To remove a phone number from a user or resource account, use the [Remove-CsPhoneNumberAssignment](Remove-CsPhoneNumberAssignment.md) cmdlet.

```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md:
```diff
@@ -117,8 +125,8 @@ Accept wildcard characters: False
-The phone number to assign to the user or resource account. Supports E.164 format like +12065551234 and non-E.164 format like 12065551234. We are also supporting
-Direct Routing numbers with extensions using the formats +1206555000;ext=1234 or 1206555000;ext=1234
+The phone number to assign to the user or resource account. Supports E.164 format like +12065551234 and non-E.164 format like 12065551234. The phone number can not have
+"tel:" prefixed. We are supporting Direct Routing numbers with extensions using the formats +1206555000;ext=1234 or 1206555000;ext=1234
```
```suggestion
"tel:" prefixed. We support Direct Routing numbers with extensions using the formats +1206555000;ext=1234 or 1206555000;ext=1234.

```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md:
```diff
@@ -134,7 +142,8 @@ Accept wildcard characters: False
 
 ### -PhoneNumberType
-The type of phone number to assign to the user or resource account. The supported values are DirectRouting, CallingPlan and OperatorConnect.
+The type of phone number to assign to the user or resource account. The supported values are DirectRouting, CallingPlan and OperatorConnect. When you acquire a phone number
```
```suggestion
The type of phone number to assign to the user or resource account. The supported values are DirectRouting, CallingPlan, and OperatorConnect. When you acquire a phone number

```

JohanFreelancer9 commented on teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md:
```diff
@@ -161,5 +170,11 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
+number. You will have to clear the phone number from the on-premises Active Directory and let that change sync into Microsoft 365 first.
+
+The previous command for assigning phone numbers to users Set-CsUser had the parameter HostedVoiceMail. Setting HostedVoiceMail for Microsoft Teams users is no longer
+necessary and that is why the parameter is not avaiable on Set-CsPhoneNumberAssignment.
```
```suggestion
necessary and that is why the parameter is not available on Set-CsPhoneNumberAssignment.

```

JohanFreelancer9 reviewed (changes requested):
Copy review - @JensTrier, kindly commit the suggested changes. Many thanks.

opbld30 commented:
Docs Build status updates of commit _[9f6e826](https://github.com/JensTrier/jenstr-office-docs-powershell/commits/9f6e826e711bddbe7982924ccf3a953f8462d526)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/PhoneNumerAssignmentUpdate/teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/teams/remove-csphonenumberassignment?view=teams-ps&branch=pr-en-us-8942) (teams-ps) |
[teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/PhoneNumerAssignmentUpdate/teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/teams/set-csphonenumberassignment?view=teams-ps&branch=pr-en-us-8942) (teams-ps) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C2%5C14%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202202140752268762-8942%5Cworkflow_report.html?sv=2020-08-04&se=2022-03-17T07%3A55%3A01Z&sr=b&sp=r&sig=CZCZQRR9s0zH2ME6QWElD%2F07pZeJ1cL7KFOv6x4pRbE%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/3603fd28-efce-f87f-e986-fc8a694fc312?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

JensTrier commented:
> Copy review - @JensTrier, kindly commit the suggested changes. Many thanks.

Thanks & Done!

opbld30 commented:
Docs Build status updates of commit _[7e91af4](https://github.com/JensTrier/jenstr-office-docs-powershell/commits/7e91af4c34f88851c5394739985cb1d785804e65)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/PhoneNumerAssignmentUpdate/teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/teams/remove-csphonenumberassignment?view=teams-ps&branch=pr-en-us-8942) (teams-ps) |
[teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/PhoneNumerAssignmentUpdate/teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/teams/set-csphonenumberassignment?view=teams-ps&branch=pr-en-us-8942) (teams-ps) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C2%5C14%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202202140752596197-8942%5Cworkflow_report.html?sv=2020-08-04&se=2022-03-17T07%3A57%3A47Z&sr=b&sp=r&sig=M6Teg8e8CF%2BXmiexYHaS%2FZmmHrrc6trX7x4c0%2Fxwn8A%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/3603fd28-efce-f87f-e986-fc8a694fc312?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

yogkumgit commented:
@dariomws @tseward @get-itips

JensTrier commented:
@yogkumgit I would like to add the following info to the Outputs section with two examples. Can I do it in this PR or should I create a new one?

On successful execution of the command no result object is present. On failed execution of the command, a result object is present with a Code string parameter to indicate the error code and a Message string parameter with the additional error or warning message.

dariomws commented:
Hi @JensTrier, you can do it in this PR while it's still open.

opbld32 commented:
Docs Build status updates of commit _[8fd085b](https://github.com/JensTrier/jenstr-office-docs-powershell/commits/8fd085b0785e3854c9d4d934268ca3014734e59b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/PhoneNumerAssignmentUpdate/teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/teams/remove-csphonenumberassignment?view=teams-ps&branch=pr-en-us-8942) (teams-ps) |
[teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/PhoneNumerAssignmentUpdate/teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/teams/set-csphonenumberassignment?view=teams-ps&branch=pr-en-us-8942) (teams-ps) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C2%5C15%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202202150841522971-8942%5Cworkflow_report.html?sv=2020-08-04&se=2022-03-18T08%3A44%3A40Z&sr=b&sp=r&sig=BJcHdYyZeWAQnkIWrB6A1RWFasQJCu1U6IHTwQPZmSg%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/3603fd28-efce-f87f-e986-fc8a694fc312?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

JensTrier commented:
I've now added text to description section in both cmdlet files and added a new example to Set-CsPhoneNumberAssignment show the result object. Please have a look

opbld31 commented:
Docs Build status updates of commit _[989b1ea](https://github.com/JensTrier/jenstr-office-docs-powershell/commits/989b1ea197ef744725848e14302720beca0d63b4)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/PhoneNumerAssignmentUpdate/teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/teams/remove-csphonenumberassignment?view=teams-ps&branch=pr-en-us-8942) (teams-ps) |
[teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/PhoneNumerAssignmentUpdate/teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/teams/set-csphonenumberassignment?view=teams-ps&branch=pr-en-us-8942) (teams-ps) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C2%5C15%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202202150844101693-8942%5Cworkflow_report.html?sv=2020-08-04&se=2022-03-18T08%3A47%3A28Z&sr=b&sp=r&sig=5hqQEkzBzHLBC1VI7%2BfURXu4GZtb%2FxkrV161ivvV6H4%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/3603fd28-efce-f87f-e986-fc8a694fc312?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

dariomws commented:
@JohanFreelancer9 @tseward @get-itips, please review the latest addition.

yogkumgit commented:
@tiburd Please merge this PR. Thanks

opbld33 commented:
Docs Build status updates of commit _[3235c19](https://github.com/JensTrier/jenstr-office-docs-powershell/commits/3235c19b819975b24d90f845eaacf13aa2199dea)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/PhoneNumerAssignmentUpdate/teams/teams-ps/teams/Remove-CsPhoneNumberAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/teams/remove-csphonenumberassignment?view=teams-ps&branch=pr-en-us-8942) (teams-ps) |
[teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md](https://github.com/JensTrier/jenstr-office-docs-powershell/blob/PhoneNumerAssignmentUpdate/teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/teams/set-csphonenumberassignment?view=teams-ps&branch=pr-en-us-8942) (teams-ps) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C2%5C15%5C3603fd28-efce-f87f-e986-fc8a694fc312%5CPullRequest%5C202202151952446420-8942%5Cworkflow_report.html?sv=2020-08-04&se=2022-03-18T19%3A55%3A01Z&sr=b&sp=r&sig=zkkvqMFALx98EVS3ni6CFNHzAlNWCYvdEvZ0D9x9TwY%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/3603fd28-efce-f87f-e986-fc8a694fc312?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>
